Automattic/wp-calypso
# Account Recovery: From step 6 to the final

## Summary
This PR is based on #13364 and #13404. It aims to chain the New Password step to the final Password Reset Succeeded page.
![demo-full-ar](https://cloud.githubusercontent.com/assets/1842898/25471618/8904e032-2b5a-11e7-952c-ddae54f32961.gif)

## Test Plan
1. Open an incognito window and make sure you are logged out.
1. Visit http://calypso.localhost:3000/account-recovery
1. Choose a test account and follow the instructions. You should be able to arrive the final "Password Reset Succeeded" page as the screencast above.
1. Click on "Log in", and try to log in with your new password. You should be able to log in with it.
